intrahost.py

```diff
@@ -293,16 +295,17 @@ def parse_alleles_string(allelesStr):
     return alleleCounts
 
 
-def get_mpileup_allele_counts(inBam, chrom, pos, inConsFasta):
+def get_mpileup_allele_counts(inBam, chrom, pos, inConsFasta, samtools=None):
     """ Return {allele : [forwardCount, reverseCount], ...}
         allele is:
             Iins for insertions where ins represents the inserted bases
             Dlen for deletions where len is the length of the deletion
             base itself for non-indels
             'i' or 'd', in which case report count for consensus.
     """
+    samtools = samtools or SamtoolsTool()
```

Comment:
Yeah, skipping the check I'd probably the way to go. The conda calls we use to check the install are the slow part. To call `conda list <package>` can take multiple seconds, even with the extra params that help reduce the time. Conda is slow.